src/components/Card/CategoryCard.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHeart, faHeartRate } from "@fortawesome/free-solid-svg-icons";
+import rupeeIcon from "../../assets/rupee.svg";
+import "./Card.css";
+import { Link } from "react-router-dom";
+import { useFilters } from "../../store/filter-context";
+import { useEffect } from "react";
```

Comment:
could import useEffect with react in single import statement

Reply:
done all the changes